helpers/htmlAttributeObserverController.js

```diff
@@ -27,12 +28,22 @@ export class HtmlAttributeObserverController {
 		controllerCallbacks.delete(this);
 	}
 
-	_handleContextChange(attributeName) {
-		if (attributeName !== this._attribute) return;
-		this.value = document.documentElement.hasAttribute(this._attribute)
-			? document.documentElement.getAttribute(this._attribute)
-			: undefined;
+	_handleContextChange(attributeNames) {
+		const attributes = attributeNames.filter(attr => this._attributes.includes(attr));
+		if (attributes.length === 0) return;
+		this._setAttributes(attributes);
 		this._host.requestUpdate();
 	}
 
+	_setAttributes(attributes) {
```

Comment:
I think this name just threw me for a loop.  Should this be named `setAttributeValues` instead?  The way I read it above, I though this was setting the attribute names that should be observed.  I think it would be good to be explicit with `attributeName` and `attributeValue` to avoid any confusion.  Or perhaps `updateAttributeValues` would be even better, since it's taking in a list of attribute names and then obtaining the value from elsewhere, as opposed to a `setter` which would be taking the values as an arg.

Reply:
Oh, good point, yes. I think this could be named to be much more clear. 👍